web/src/utils/dateFormat.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Replace T and remove milliseconds
```

Comment:
This doesn't fully describe the util function. Add some information like `from datetime format` at the end.